front/app/modules/commercial/representativeness/admin/containers/Dashboard/Header.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+
+// components
+import { Title, Box, Text } from '@citizenlab/cl2-component-library';
+
+// i18n
+import messages from './messages';
+import { FormattedMessage } from 'utils/cl-intl';
+
+const Header = () => (
+  <Box mb="36px">
+    <Title variant="h1">
+      <FormattedMessage {...messages.pageTitle} />
+    </Title>
+    <Box>
+      <Text color="label">
+        <FormattedMessage
+          {...messages.pageDescription}
+          values={{
+            representativenessArticleLink: (
+              <a
+                href="https://en.wikipedia.org/wiki/Chi-squared_test"
```

Comment:
This will become a link to a support article, still gotta ask Product what to put in here for now :)